### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by wcatz**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3914,5 +3914,16 @@
...
+  },
+    {
+    "ticker": "OTG",
+    "poolId": "c825168836c5bf850dec38567eb4771c2e03eea28658ff291df768ae",
+    "social": {
+      "website": "https://adamantium.online/",
+      "twitter": "OTG_WCat"
+    },
+    "telegram": "WCat13",
+    "operatorName": "WCat",
```
Comment (Bez625) [inline registry.json:3926]: Please can you change this to either your real full name, real first name or leave blank?

File: registry.json [hunk 2]
```diff
@@ -3914,5 +3914,16 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+    {
+    "ticker": "OTG",
+    "poolId": "c825168836c5bf850dec38567eb4771c2e03eea28658ff291df768ae",
+    "social": {
+      "website": "https://adamantium.online/",
+      "twitter": "OTG_WCat"
```
Comment (Bez625) [inline registry.json:3923]: Please can you add the missing facebook, instagram and youtube values (even if blank)

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by wcatz** [comment]
Done thank you

File: registry.json [hunk 3]
```diff
@@ -3915,6 +3915,17 @@
     "operatorName": "",
     "github": "rafek47"
   },
+    {
+    "ticker": "OTG",
+    "poolId": "c825168836c5bf850dec38567eb4771c2e03eea28658ff291df768ae",
+    "social": {
+      "website": "https://adamantium.online/",
+      "twitter": "OTG_WCat"
```
Comment (Bez625) [inline registry.json:3923]:
```suggestion
      "twitter": "OTG_WCat",
      "facebook": "",
      "instagram": "",
      "youtube": ""
```

File: registry.json [hunk 4]
```diff
@@ -3915,6 +3915,20 @@
...
+      "website": "https://adamantium.online/",
+      "twitter": "OTG_WCat",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "WCat13",
+    "operatorName": "Wayne",
+    "github": "wcatz"
+
```
Comment (Bez625) [inline registry.json:3931]:
```suggestion
    },
```

**Review by Bez625 — APPROVED** [review]

**Pull request merged**